runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -1009,7 +1013,9 @@ inlineIntrinsicIndexOf(TR::Node * node, TR::CodeGenerator * cg, bool isLatin1)
 
    generateS390LabelInstruction(cg, TR::InstOpCode::LABEL, node, fullVectorLabel);
 
-   generateRIEInstruction(cg, TR::InstOpCode::CIJ, node, length, cg->machine()->getVRFSize(), failureLabel, TR::InstOpCode::COND_BL);
+   generateS390CompareAndBranchInstruction(cg, TR::InstOpCode::C, node,
+                                           length, static_cast<int32_t>(cg->machine()->getVRFSize()),
+                                           TR::InstOpCode::COND_BL, failureLabel);
```

Comment:
`static_cast` here seems unnecessary, but also we already cache the value of `cg->machine()->getVRFSize()` into `sizeOfVector` so we should be just reusing that variable.